gobblin-yarn/src/main/java/org/apache/gobblin/yarn/YarnService.java

```diff
@@ -159,6 +173,11 @@
 
   private volatile boolean shutdownInProgress = false;
 
+  // The number of containers requested
```

Comment:
Need a bit more documentation on this. Is this the current outstanding number of containers requested based on the last call to requestContainers .. or the total lifetime number..